pom.xml

```diff
@@ -532,6 +521,9 @@
     <profiles>
         <profile>
             <id>release</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Maybe this is why gpg was brought in.  release profile does not needs to be activated by default.  Does it require you have to a default profile?